src/views/ChooseObject.vue

```diff
@@ -121,19 +122,37 @@ export default class ChooseObject extends Vue {
     let name = objects[index].name;
     const data = { selection: name };
 
-    const ok = await Swal.fire({
-      title: "Your selection",
-      text: "You have selected " + name,
-      reverseButtons: true,
-    });
-
-    if (ok) {
-      await post<any>("/api/select", data);
-      this.yourSelection = ObjectClass.getImage(objects[index]);
-      this.characterSelected = true;
-      this.displayCommand = false;
-      this.guess();
+    let userGuess = "";
+
+    if(!this.characterSelected) {
+      const ok = await Swal.fire({
+        title: "Your selection",
+        text: "You have selected " + name,
+        reverseButtons: true,
+      });
+
+      if (ok) {
+        await post<any>("/api/select", data);
+        this.yourSelection = ObjectClass.getImage(objects[index]);
+        this.characterSelected = true;
+        this.displayCommand = false;
+        this.guess();
+      }
+    } else {
+      userGuess = await Swal.fire({
+        title: `Are you sure "${name}" is the correct user?`,
+        text: 'If you click "Cancel", continue with questions.',
```

Comment:
Even better would be instead of a Cancel button you had a "Continue" button. The text could be "Click 'Continue' to ask more questions". Clicking on the "Continue" button would then immediately ask the next computer question. Also, I think the "Confirm" button should be red (as a warning) and "Continue" can then be blue.